src/System.Private.ServiceModel/tools/CertificateGenerator/App.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <add key="testserverbase" value=""/>
+    <add key="CertExpirationInDay" value="30"/>
+    <add key="CrlFileLocation" value="c:\wcftest\test.crl"/>
+  </appSettings>
```

Comment:
Not blocking -- just recording the use of another hard-code path for later consideration.